fe/fe-core/src/main/java/com/starrocks/common/proc/TableProcDir.java

```diff
@@ -86,15 +86,15 @@ public ProcNodeInterface lookup(String entryName) throws AnalysisException {
         }
 
         if (entryName.equals(PARTITIONS)) {
-            if (table.getType() == TableType.OLAP) {
+            if (table.getType() == TableType.OLAP || table.getType() == TableType.MATERIALIZED_VIEW) {
```

Comment:
I think we could add a method to unify the OLAP and MATERIALIZED_VIEW type.  such as isNativeTable?

Reply:
ok